2017-10/oct-25.md

```diff
@@ -0,0 +1,38 @@
+# Babel Weekly Notes - 2017-10-25
+
+## News
+
+> Logan/Henry were at Chrome Dev Summit! (https://twitter.com/left_pad/status/922507305874104320)  
+> Sven at ReactiveConf (https://twitter.com/svensauleau/status/923210175011590146)  
+> Added [@azz](https://github.com/azz), [@keithamus](https://github.com/keithamus), [@Andarist](https://github.com/Andarist) to the team
+> Also added [@bmeurer](https://github.com/bmeurer) (other v8 members are already collaborators) since already made like 5 perf PRs (https://github.com/babel/babel/pull/6581)
```

Comment:
They are missing on the team page. I made https://github.com/babel/website/issues/1444